src/views/GRTSystemIntakeReview/index.tsx

```diff
@@ -19,7 +19,12 @@ export const GRTSystemIntakeReview = () => {
   return (
     <div>
       <Header />
-      <div className="grid-container">
+      <main
```

Comment:
This `main` tag seems to have a lot of consistent props passed to it, which are pretty important for our app. Should we make this a light-weight component that renders children passed to it? That way we don't forget to add z-index, etc, as we add new pages to the app.

Reply:
I agree. I was unsure whether to split it out or not. Let me give a try at consolidating that into a reusable component.

Reply:
@sarboc I broke out that `main` tag into its own component.